### PR TITLE
Add integration tests

### PR DESCRIPTION
**Pull request by NilashishC**
Signed-off-by: NilashishC <nilashishchakraborty8@gmail.com>
Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/1367

##### SUMMARY
- Add integration tests

### CODE REVIEW

**Review by ashwini-mhatre — APPROVED** [review]

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

**Comment by goneri** [comment]
recheck

**Comment by goneri** [comment]
recheck

**Comment by goneri** [comment]
recheck

**Comment by goneri** [comment]
recheck

**Comment by goneri** [comment]
recheck

**Comment by NilashishC** [comment]
recheck

**Comment by goneri** [comment]
recheck

**Comment by NilashishC** [comment]
recheck

**Comment by NilashishC** [comment]
@goneri it is now failing with an unrelated error? Could you please help?

**Comment by goneri** [comment]
recheck

**Comment by goneri** [comment]
recheck

File: zuul.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (goneri) [inline zuul.yaml:1]: Please, remove this file from your PR.

**Review by goneri — CHANGES_REQUESTED** [review]

**Comment by GomathiselviS** [comment]
recheck

**Pull request closed without merging**